lib/ansible/modules/iptables.py

```diff
@@ -32,15 +31,22 @@
     platform:
         platforms: linux
 notes:
-  - This module just deals with individual rules. If you need advanced
-    chaining of rules the recommended way is to template the iptables restore
-    file.
+  - This module deals with compiling iptables policies from iptables rules.
+    One off iptables manipulation can be dangerous, but is also slow.
 options:
+  do:
```

Comment:
no, nope, don't do this, a 'do' option is not something we would accept

if you want to add a persistent=no|yes option to save or not, that would be acceptable